_posts/2020-05-13-cockpit-219.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cockpit 219
+author: kkoukiou
+date: 2020-05-13
+tags: cockpit linux
+slug: cockpit-219
+category: release
+summary: Cockpit with XYZ
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 219.
+
+### Logs: Improved journal logs filtering
+
+It's now possible to filter logs by providing keywords in a text filter which may include the unit of interest, time constraints, priority and [journal fields].
+There is also the possibility to pause/resume the logs streaming.
+Lastly journal logs which appear in other pages are now linked to the the full journal in the Logs page.
+
+![Journal Logs Filtering](/images/journal-logs-filtering.png)
+![Journal Logs Per Page Link](/images/journal-logs-per-page-link.png)
```

Comment:
I'm not sure what this second screenshot should convey. It probably misses some context, or at least an explanation?